app/views/reductions.html

```diff
@@ -0,0 +1,105 @@
+{% extends "includes/layout.html" %} 
+
+{% block content %}
+<main id="content" role="main">
+    {% include "includes/breadcrumbs.html" %}
+    
+    {% include "includes/validation-error-messages.html" %} 
```

Comment:
Question: What is validation-error-messages used for?  Seen it in a load of different places

Reply:
See answer to comment above.